### ISSUE TITLE
Forbid competitors from looking at other cubes (?)

### ISSUE THREAD

**Issue by pedrosino** [thewca/wca-regulations#307]
From the China Championship report.

I believe this is something that should be "forbidden" already by common sense and sportsmanship, but that way we cannot do anything if people break this, so maybe including it in the regulations is a good idea.

There are 2 main ways of attempting to get information from peeking at other's cubes:
1 - competitors in the waiting area trying to watch other solves
This is harder to accomplish, and depends a lot on the waiting and competing areas setup.

2 - competitor sitting right next to another and watching the inspection/solve
This is easy to avoid if you are a competitor (just look away), and probably easier to catch by judge/delegate. Also, this is more serious since you're closer to the cube and can see more information more easily.

Maybe add
2g4) Competitors in the Competitors Area must not try to see other competitors' solves. Penalty: disqualification of the competitor from the event, at the discretion of the WCA Delegate.

A3b3) The competitor must not watch other competitors' solves after being called and before starting their attempt.

**Comment by Laura-O** [comment]
I don't think your suggestions actually express what they should and they are way to imprecise.
1. We don't want to forbid competitors to look or watch other competitors' attempts. What we want is that they do not have any **advantage** by looking at a scrambled puzzle or a solution of another competitor.
2. This is only relevant for attempts that the (looking) competitor hasn't done yet, so this should be mentioned as well.

However, I feel very uncomfortable with such a regulation:
1. This is always a problem with regulations "at the discretion of the delegate", but judging if someone is looking at other competitors is really, really difficult. You can hardly tell if he is just looking around or intentionally inspecting what the other one is doing. Having such a regulation actually means that it's legal to disqualify everyone who is turning around his head left or right at the solving station.
2. I'm sure that it will lead to a lot of complaints from the community. It will change, how people behave in the competitors area and before/after they are doing their attempts because of the aforementioned situation. And I completely understand it. I can remember several situations when I just looked at other competitors while they were doing attempts that I hadn't done and I never had the intention to get any information about the scramble or solution.

I'm not completely sure about this yet, but I just had the idea that it might be useful to add a generic regulation that can be used to punish competitors for any kind of behaviour that gives them an advantage. We have 2k2 now (_"Is suspected of cheating or defrauding the officials during the competition."_), but "cheating" is a strong word, which I wouldn't use in these situations.
Also, avoiding to mention "looking at other competitors' attempts" takes away some pressure from delegates that they have to observe, where every competitor is looking at. But as said before: just a spontaneous idea, not ready to print. :-)

**Comment by pedrosino** [comment]
Yeah, I was still sleepy and not thinking very well. It should mention the "intention of gaining advantage" or "acquiring information about scrambled states"

**Comment by pedrosino** [comment]
Given the recent discussion about Virginia Open Spring 2017, I re-read this and think I might have a better proposal:

2g3) Competitors in the Competitors Area must not attempt to obtain information about scrambles they have not attempted yet (e. g. by communicating with each other about the scrambled states of the puzzles of the round in progress or by watching other competitors solving). Penalty: disqualification of the competitor from the event, at the discretion of the WCA Delegate.

**Comment by lgarron** [comment]
> 2g3) Competitors in the Competitors Area must not attempt to obtain information about scrambles they have not attempted yet (e. g. by communicating with each other about the scrambled states of the puzzles of the round in progress or by watching other competitors solving). Penalty: disqualification of the competitor from the event, at the discretion of the WCA Delegate.

I think that's pretty reasonable, but it's still unsatisfactory. (Previously, 2g3 just said "communicate", which made it clear that receiving information is not allowed even if you didn't "attempt" it.)

I think I'd be more comfortable with it if we also added a Regulation or Guideline saying that a competitor who believes they have unintentionally received significant information about scrambles should report it to the Delegate immediately (or at least before their next attempt).
("Significant" is a bit of a cop-out, though. I'm trying to imply that vague things like "the crowd cheered because Feliks got a good time on the scramble" is out of scope. But I think it would be hard to judge, say, overhearing something like a competitor saying "easy cross" to another competitor who's also done the same scramble.)
